tensorflow_datasets/image/imagewoof.py

```diff
@@ -0,0 +1,111 @@
+The breeds are: Australian terrier, Border terrier, Samoyed, Beagle, Shih-Tzu, English foxhound, Rhodesian ridgeback,
+Dingo, Golden retriever, Old English sheepdog.
+"""
+
+lbl_dict = {
+    'n02093754': 'Australian terrier',
+    'n02089973': 'Border terrier',
+    'n02099601': 'Samoyed',
+    'n02087394': 'Beagle',
+    'n02105641': 'Shih-Tzu',
+    'n02096294': 'English foxhound',
+    'n02088364': 'Rhodesian ridgeback',
+    'n02115641': 'Dingo',
+    'n02111889': 'Golden retriever',
+    'n02086240': 'Old English sheepdog'
+}
+
+
+class Imagewoof(tfds.core.GeneratorBasedBuilder):
+
+    # {TODO}: Set up version.
+    VERSION = tfds.core.Version('0.1.0')
+
+    def _info(self):
+        return tfds.core.DatasetInfo(
+            builder=self,
+            description=_DESCRIPTION,
+            # tfds.features.FeatureConnectors
+            features=tfds.features.FeaturesDict({
+                "image":
+                tfds.features.Image(),
+                # Here, labels can be one of 3 classes
+                "label":
+                tfds.features.ClassLabel(
+                    names=['Australian terrier', 'Border terrier', 'Samoyed', 'Beagle', 'Shih-Tzu', 'English foxhound',
+                           'Rhodesian ridgeback', 'Dingo', 'Golden retriever', 'Old English sheepdog']),
+            }),
+            supervised_keys=("image", "label"),
+            homepage="https://github.com/fastai/imagenette",
+            citation=_CITATION
+        )
+
+    def _split_generators(self, dl_manager):
+        # {TODO}: Downloads the data and defines the splits
+        # dl_manager is a tfds.download.DownloadManager that can be used to
+        # download and extract URLs
+        extracted_path = dl_manager.download_and_extract(_IMAGEWOOF_URL) 
+        extracted_path = os.path.join(extracted_path, 'imagewoof2')
+        # Specify the splits
+        return [
+            tfds.core.SplitGenerator(
+                name=tfds.Split.TRAIN,
+                gen_kwargs={
+                    "images_dir_path": os.path.join(extracted_path, "train"),
+                }),
+            tfds.core.SplitGenerator(
+                name=tfds.Split.VALIDATION,
+                gen_kwargs={
+                    "images_dir_path": os.path.join(extracted_path, "val"),
+                }),
+        ]
+
+    def _generate_examples(self, images_dir_path):
+        # {TODO}: Yields (key, example) tuples from the dataset
+        print(images_dir_path)
+        count = 0
+        for image_folder in tf.io.gfile.listdir(images_dir_path):
+            for image_file in tf.io.gfile.listdir(os.path.join(images_dir_path, image_folder)):
+                yield count, {
+                    'image': '{}/{}/{}'.format(images_dir_path, image_folder, image_file),
+                    'label': lbl_dict[image_folder]
+                }
+                count += 1
```

Comment:
Add extra line space and please run `./oss_scripts/lint.sh tensorflow_datasets/image` and make sure score is10